public/src/components/PostItem/CommentsSection.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import CommentComponent from './CommentComponent';
+import methods from '../../utils/methods';
+
+const { sortComments } = methods;
+
+export default class CommentsSection extends React.Component {
+    constructor(props) {
+        super(props);
+        this.commentToShowStep = 20;
+        this.getComments = this.getComments.bind(this);
+        this.showMoreComments = this.showMoreComments.bind(this);
+        this.state = {
+            commentsToShow: this.commentToShowStep,
+        };
+    }
+
+    getComments() {
+        const { commentsToShow } = this.state;
+        const { comments } = this.props;
+        let sortedComments = sortComments([...comments]);
+        sortedComments = sortedComments
+            .filter(comment => comment.body !== 'null')
+            .slice(0, commentsToShow);
+
+        if (sortedComments) {
+            return sortedComments.map((comment, index) => (
+                <CommentComponent key={index}
```

Comment:
`key={index}` is ok, but maybe there's some unique identifier from reddit in comment json?

Reply:
fixed https://github.com/YuriyBC/Reddit-archive/commit/53304bdd6d6b8a046cd5b9e5897380874253994f